Assets/Dependencies/SmashBrew/Stage/StageLedge.cs

```diff
@@ -1,26 +1,53 @@
 using HouraiTeahouse.SmashBrew.Characters;
 using UnityEngine;
+using UnityEngine.Networking;
 
 namespace HouraiTeahouse.SmashBrew.Stage {
 
     [AddComponentMenu("SmashBrew/Stage/Ledge")]
-    public class StageLedge : MonoBehaviour {
+    public class StageLedge : NetworkBehaviour {
 
         [SerializeField]
         bool _direction;
 
-        public bool Occupied { get; private set; }
+        public bool Direction
+        {
+            get { return _direction; }
+        }
+
+        [SyncVar, SerializeField]
+        bool _occupied;
+
+        public bool Occupied {
+            get { return _occupied; }
+            private set { _occupied = value; }
```

Comment:
This should include a `isServer` check to assure that the value is properly propagated to clients via the SyncVar, otherwise the value could be alltered locally and not properly propagated.

It may be a good idea to create a utility function for this.